### PR TITLE
Adds arabian cat + exotic cats crate

### PR DESCRIPTION
**Pull request by Archanial**
## About The Pull Request

Halal cat, when he sits he prays to Allah.
Sprite made by Kate, sitting and dead sprite made by my friend

Also exotic cat crate which gives you random rare cat

## Changelog
:cl:
add: Arabian cat
add: Exotic cat crate
/:cl:

### CODE REVIEW

**Comment by r1ks-iwnl** [comment]
inshallah

**Comment by ItsTisme** [comment]
ok he do be praying tho :flushed:

**Comment by Nosnek199** [comment]
post sprite

**Comment by Archanial** [comment]
Use diff bot, you can find it with rest of the actions bellow pr.

**Comment by TeomanTheGreat** [comment]
Speedmerge immediately

**Comment by Kmc2000** [comment]
good PR

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -2180,6 +2180,28 @@
...
+	name = "Exotic Cat Crate"
+	desc = "Commes with one of the exotic cats, collar and a toy."
+	cost = 5500
+	contains = list(/obj/item/clothing/neck/petcollar,
+					/obj/item/toy/cattoy)
+	crate_name = "cat crate"
+
+/datum/supply_pack/critter/cat/exotic/generate()
+	. = ..()
+	switch(rand(5))
```
Comment (utilbird) [inline code/modules/cargo/packs.dm]:
```suggestion
	switch(rand(1, 5))
```

**Review by utilbird — APPROVED** [review]

**Pull request merged**